docker/populate/kolibri-pick-content-from-channel

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python3
```

Comment:
Can't we fetch this file from the original repository? Instead of copying it and the 99% chance that it will get outdated as soon as the first change.